src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,22 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
+import core.basesyntax.figures.Figure;
+import core.basesyntax.supplier.FigureSupplier;
+
 public class HelloWorld {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[5];
+        for (int i = 0; i < figures.length; i++) {
+            if (i == 2) {
+                figures[i] = figureSupplier.getDefaultFigure();
+            } else {
+                figures[i] = figureSupplier.getRandomFigure();
+            }
```

Comment:
this is invalid - not first half being created with default figures, but only one figure with index 2. rewrite using dynamic middle-index calculation